front/src/components/mmt-locale-switcher/mmt-locale-switcher.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="locale-switcher">
+  <v-select :items="locales"
+            v-on:change="switchLocale"
+            :label="$t('footer.language')"
+            v-model="this.$root.$i18n.locale">
```

Comment:
no, no, no))

Reply:
please, place state of localization into store (vuex)
and in mmt-locale-switcher use store, not root

Reply:
I have saved language in store (vuex)